docs/api/rest_api/rest_api_reference/input/ibexa-segments.raml

```diff
@@ -18,6 +70,79 @@ displayName: Segments
                     application/vnd.ibexa.api.Segment+json:
                         type: SegmentWrapper
             401:
-                description: Error - the user is not authorized to load this group's segments.
+                description: Error - the user is not authorized to load this segment.
             404:
-                description: Error - the identifier doesn't match any segment group.
+                description: Error - the identifier doesn't match any segment.
+/{id}:
+    patch:
+        displayName: Update segment data
+        description: Updates the segment's data.
+        headers:
+            Content-Type:
+                description: The SegmentUpdate schema encoded in XML or JSON format.
+                example: |
+                    application/vnd.ibexa.api.SegmentUpdate+xml
+                    application/vnd.ibexa.api.SegmentUpdate+json
+            Accept:
+                description: A segment is returned in XML or JSON format.
+                example: |
+                    application/vnd.ibexa.api.Segment+xml
+                    application/vnd.ibexa.api.Segment+json
+        body:
+            application/vnd.ibexa.api.SegmentUpdate+xml:
+                type: SegmentUpdate
+                example: !include examples/segments/id/PATCH/SegmentUpdate.xml.example
+            application/vnd.ibexa.api.SegmentUpdate+json:
+                type: SegmentUpdateWrapper
+                example: !include examples/segments/id/PATCH/SegmentUpdate.json.example
+        responses:
+            200:
+                description: OK- returns updated segment's data.
+                body:
+                    application/vnd.ibexa.api.Segment+xml:
+                        type: Segment
+                        example: !include examples/segments/id/PATCH/Segment.xml.example
+                    application/vnd.ibexa.api.Segment+json:
+                        type: SegmentWrapper
+                        example: !include examples/segments/id/PATCH/Segment.json.example
+            400:
+                description: Error - input data validation failed.
+                body:
+                    application/vnd.ibexa.api.ErrorMessage+xml:
+                        type: ErrorMessage
+                    application/vnd.ibexa.api.ErrorMessage+json:
+                        type: ErrorMessage
+            401:
+                description: Error - the user is not authorized to create segments.
+                body:
+                    application/vnd.ibexa.api.ErrorMessage+xml:
+                        type: ErrorMessage
+                    application/vnd.ibexa.api.ErrorMessage+json:
+                        type: ErrorMessage
+            406:
```

Comment:
Same here, I would mention the example when this can happen (not unique identifier)